packages/server/beaver_triple_service/utils/interceptor.go

```diff
@@ -35,33 +35,33 @@ func BtsAuthFunc(ctx context.Context) (context.Context, error) {
 }
 
 func getSecret() ([]byte, error) {
-	raw, ok := os.LookupEnv("JWT_SECRET_KEY")
-	if !ok {
-		return nil, status.Error(codes.Internal, "jwt auth key is not provided")
-	}
-	secret, err := base64.StdEncoding.DecodeString(raw)
```

Comment:
confirmed.  
This is a just comment.
I think it is better to use base64 to make the best use of 256 bit as a secret key.